### ISSUE TITLE
Better error message for no host or client.conf.string

### ISSUE THREAD

**Issue by javier** [questdb/kafka-questdb-connector#20]
I had a typo in which I was passing the `client.conf.string=` property, with a trailing equal sign. Of course I had an error message saying this

```
{"error_code":500,"message":"Either 'client.conf.string' or 'host' must be set."
```

I misinterpreted the message and I thought I was passing both the `client.conf.string` and `host` properties, and I was trying to see why host was being used. After a while I noticed my error. This might be only me, but maybe rewording the message to say that None of the properties have been configured would make things easier.

Thanks!

**Comment by jerrinot** [comment]
@javier: I am considering to not talk about `host` at all. Since new deployments should use the client conf string anyway. WDYT?
